client/lib/directly/index.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+// import { loadScript } from 'lib/load-script';
+
+const DIRECTLY_SCRIPT_PATH = 'https://www.directly.com/widgets/rtm/embed.js';
+const DEFAULT_RTM_WIDGET_OPTIONS = {
+	// Automattic widget ID
+	id: '8a2968fc57d1e2f40157f42bf2d43160',
+	// By default don't show the "Ask a Question" button
+	displayAskQuestion: false
+};
+
+let initialized = false;
+
+const loadDirectlyScript = ( callback = noop ) => {
```

Comment:
@omarjackman I've not really dabbled much in third-party dependencies so much as bundling techniques, but I'll drop a few thoughts:

>a nicety of linking to the remote script is we can do nothing and receive changes (bugfixes, new features, etc) from Directly

Could also receive nice new security vulnerabilities from a compromised script 😬 

>I'm also curious how this affects the larger conversation happening now about performance in Calypso?

If it were imported in only one section's bundle (or a "sub-bundle" via `AsyncLoad` / `asyncRequire`) it shouldn't really be an issue for the larger application.

Kinda a shame that the script performs actions / side-effects immediately on load. The script looks pretty small... I've not fully digested the rest of the conversation of this pull request, but would it be possible to simply recreate the behavior of whatever it is Directly is doing? (Aside: I get a 401 "Unauthorized" from Directly when visiting `/help/contact`, not sure how to test this)